File: 1.go
package main

import (
	"fmt" // пакет для вывода данных
)

func main() {
	var N int // объявление переменной 
	fmt.Scan(&N) // ввод с консоли
	for i := N; i >= 0; i-- {
		for letter := 'Z' - (26 - N); letter >= 'A'+ i; letter-- { // перебор алфавита
			fmt.Printf("%c ", letter) // %с - вывод символа
		}
		fmt.Println()
	}
}

File: 1.kt
import java.util.Scanner // входные данные

fun main() {
    val scanner = Scanner(System.`in`)
    val N = scanner.nextInt()

    for (i in N downTo 0) { // downTo - обратный порядок перебора i от N до 0  
        for (letter in 'Z' - (26 - N) downTo 'A' + i) { // перебор алфавита
            print(letter + " ")
        }
        println()
    }
}
